package.json

```diff
@@ -1,6 +1,7 @@
 {
 	"name": "typobot",
 	"dependencies": {
+		"abort-controller": "^3.0.0",
```

Comment:
why?

Reply:
IDK. It said it couldn't find the module

Reply:
Thats weird.